packages/services/src/leaderboard.service.ts

```diff
@@ -71,14 +71,12 @@ export class LeaderboardService {
    * @param params Pagination parameters
    * @returns Object with paginated agent metrics and total count
    */
-  private async getGlobalAgentMetrics(params: {
-    limit: number;
-    offset: number;
-  }): Promise<{
+  private async getGlobalAgentMetrics(params: LeaderboardParams): Promise<{
```

Comment:
the method that calls `getGlobalAgentMetrics` is directly passing `LeaderboardParams`, so we can just reuse it